### PR TITLE
Adding filebeat role to remove duplicate logic

### PR DESCRIPTION
**Pull request by jokvedaras**
This new role will also be used by clients to integrate external sensors into the RockNSM framework.

### CODE REVIEW

**Comment by dcode** [comment]
@jokvedaras, we're reviewing this. I like the refactor approach, I just want to get a chance to test it out with our new test harness.

**Comment by dcode** [comment]
We're still working out the bugs in our CI, as it's not publicly accessible at the moment. Here's the output:

```
--> Scenario: 'single-node'
--> Action: 'lint'
--> Executing Yamllint on files found in /tmp/build/801f9bf3/pr-source/...
    /tmp/build/801f9bf3/pr-source/roles/fsf/handlers/main.yml
      3:1       error    too many blank lines (1 > 0)  (empty-lines)
    
    /tmp/build/801f9bf3/pr-source/roles/fsf/meta/main.yml
      9:1       error    too many blank lines (1 > 0)  (empty-lines)
    
    /tmp/build/801f9bf3/pr-source/roles/suricata/meta/main.yml
      9:1       error    too many blank lines (1 > 0)  (empty-lines)
    
An error occurred during the test sequence action: 'lint'. Cleaning up.
```

I'll make notes in the review, but dropping here for reference. Needs to pass `yamllint` and `ansible-lint` before we can run it.

File: roles/fsf/handlers/main.yml [hunk 1]
```diff
@@ -1,7 +1,3 @@
 ---
-# handlers file for suricata
+# handlers file for fsf
```
Comment (dcode) [inline roles/fsf/handlers/main.yml]: Delete line 3 to appease `yamllint`.

**Comment by jokvedaras** [comment]
@dcode Done!

**Pull request merged**